docker/dockerbuild/Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM circleci/python:2.7.17-stretch
+FROM circleci/python:3.10.0
```

Comment:
this should not be 3.10.1?

Reply:
they didnt have it when i made those changes, ill update now